suites/nautilus/upgrades/concurrent_execution.yaml

```diff
@@ -0,0 +1,81 @@
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible install rhcs 4.x from cdn
+      polarion-id: CEPH-83571467
+      module: test_ansible.py
+      config:
+        use_cdn: True
+        build: '4.x'
+        ansi_config:
+          ceph_origin: repository
+          ceph_repository: rhcs
+          ceph_repository_type: cdn
+          ceph_rhcs_version: 4
+          ceph_stable_release: nautilus
+          osd_scenario: lvm
+          osd_auto_discovery: False
+          ceph_docker_registry_auth: false
+          dashboard_enabled: false
+          copy_admin_key: True
+          ceph_conf_overrides:
+            global:
+              osd_pool_default_pg_num: 128
+              osd_default_pool_size: 2
+              osd_pool_default_pgp_num: 128
+              mon_max_pg_per_osd: 4096
+              mon_allow_pool_delete: True
+            client:
+              rgw crypt require ssl: false
+              rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+                testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+      desc: test cluster 4.x cdn setup using ceph-ansible
+      abort-on-fail: True
+   - test:
+       name: Concurrent run
+       module: test_concurrent.py
+       desc: Running tests concurrently
+       concurrent_tests:
```

Comment:
IMO, we don't need to prefix `_tests` to `concurrent`... 
Does `parallel` sound better?